### ISSUE TITLE
1.2.2 --version output wrong

### ISSUE THREAD

**Issue by simonwydooghe** [lukasmartinelli/hadolint#86]
Minor 'issue'. I checked out v1.2.2 and built that tag. This is the --version output:
```
❯ hadolint --version
Haskell Dockerfile Linter v1.2.1
```

**Comment by zemanlx** [comment]
@simonwydooghe fixed by #97. If you build it now from master you should see something like this

```bash
$ hadolint --version
Haskell Dockerfile Linter v1.2.2-20-gbb0a49b
```

**Comment by simonwydooghe** [comment]
@zemanlx Cheers, thanks.
